### PR TITLE
[refactor] Player Packet Manager

### PR DESCRIPTION
**Pull request by Xiaro**
Cleaned up code, rewrote hotbar spoof, separated hotbar spoof from player packet manager

### CODE REVIEW

File: src/main/kotlin/org/kamiblue/client/module/modules/combat/CrystalAura.kt [hunk 1]
```diff
@@ -240,8 +246,13 @@ internal object CrystalAura : Module(
...
             if (it.phase == TickEvent.Phase.END) {
-                if (inactiveTicks > 5 || getHand() == EnumHand.OFF_HAND) PlayerPacketManager.resetHotbar()
-                if (inactiveTicks > 20) resetRotation()
+                if (getHand() == EnumHand.OFF_HAND) {
+                    resetHotbar()
+                }
+                if (inactiveTicks > 20) {
+                    resetHotbar()
+                    resetRotation()
+                }
```
Comment (l1ving) [inline src/main/kotlin/org/kamiblue/client/module/modules/combat/CrystalAura.kt:255]: Are you supposed to be resetting hotbar twice?

Comment (Xiaro) [reply src/main/kotlin/org/kamiblue/client/module/modules/combat/CrystalAura.kt:255]: HotbarManager will handle that itself

**Review by l1ving — APPROVED** [review]

**Comment by l1ving** [comment]
Fix merge conflicts please.

File: src/main/java/org/kamiblue/client/mixin/client/MixinMinecraft.java [hunk 2]
```diff
@@ -89,19 +86,26 @@ public void runGameLoopEnd(CallbackInfo ci) {
...
 
         ci.cancel();
 
         for (EnumHand enumhand : EnumHand.values()) {
-            ItemStack itemstack = this.player.getHeldItem(enumhand);
-            if (itemstack.isEmpty() && (this.objectMouseOver == null || this.objectMouseOver.typeOfHit == RayTraceResult.Type.MISS)) {
-                net.minecraftforge.common.ForgeHooks.onEmptyClick(this.player, enumhand);
+            ItemStack itemstack = player.getHeldItem(enumhand);
+            if (itemstack.isEmpty() && (objectMouseOver == null || objectMouseOver.typeOfHit == RayTraceResult.Type.MISS)) {
+                net.minecraftforge.common.ForgeHooks.onEmptyClick(player, enumhand);
```
Comment (l1ving) [inline src/main/java/org/kamiblue/client/mixin/client/MixinMinecraft.java:105]: Why are we not importing ForgeHooks instead?

**Review by l1ving — APPROVED** [review]

**Review by l1ving — APPROVED** [review]

**Pull request merged**